shared/networking/1.X.md

```diff
@@ -0,0 +1,159 @@
+# Network Setup
+
+* [Introduction](#introduction)
+* [Network Requirements](#network-requirements)
+* [Ethernet Connections](#ethernet-connections)
+* [Wifi Connections](#wifi-connections)
+* [Changing your network configuration](#changing-your-network-configuration)
+* [3G or Cellular Connections](#3g-or-cellular-connections)
+* [Captive Portal Network Setup](#captive-portal-network-setup)
+
+## Introduction
+
+In order to deploy code to your device(s), they will need to be connected to the internet in some shape or form. The resin.io HostOS manages your devices' network connection with [ConnMan][connman], an open source connection manager.
+
+Currently all resin.io devices are configured to favour [ethernet](#ethernet-connections) and will automatically use this connection type if it is available. You can define all manner of network configurations using [Connman][connman], for example [setting a static IP](#set-static-ip).
+
+## Network Requirements
+
+In order for a resin.io device to get outside of the local network and connect to the resin.io API, there are a few core network requirements.
+
+Resin.io makes use of the following ports:
+
+* `443` TCP - This is the most fundamental requirement - it is used to connect to the VPN and the web terminal, and many web endpoints using TLS (https://.)
+* `123` UDP - For NTP time synchronisation.
+* `53` UDP - For DNS name resolution.
+
+Each of these should work with outward only (and inward once outward connection established) firewall settings.
+
+Additionally, if the network your device is connecting to works with whitelisting, you should whitelist the following domains on Port `80` and `443`:
+* `*.resin.io`
+* `*.pubnub.com`
+
+Additionally make an outgoing connection to `mixpanel.com`, but this is not a functional requirement for resin.io, but rather allows us to track some useful metrics.
```

Comment:
yeah, people are pretty paranoid about their devices reaching out and connecting to random servers that they don't know about.